tests/manage/mcg/test_bucket_creation.py

```diff
@@ -12,40 +12,41 @@
 
 @filter_insecure_request_warning
 @aws_platform_required
-@acceptance
-@tier1
+@pytest.mark.parametrize(
+    argnames="amount,interface",
+    argvalues=[
+        pytest.param(
+            *[3, 'S3'],
+            marks=[pytest.mark.polarion_id("OCS-1298"), tier1, acceptance]
+        ),
+        pytest.param(
+            *[3, 'CLI'], marks=[tier1, acceptance, noobaa_cli_required]
+        ),
+        pytest.param(
+            *[3, 'OC'], marks=[tier1, acceptance]
+        ),
+        pytest.param(
```

Comment:
Is this a new test case?
Polarion_id?

Reply:
I wouldn't consider it a new test case - it's just a more varied and thorough parametrization of the same test. I don't think a separate Polarion TC is needed

Reply:
polarion_id seems to be deleted below.
@MeridianExplorer I suggest we create separate polarion test cases for the different amounts. The different interfaces don't need separate test cases

Reply:
@waynesun09 - Can you please create the appropriate Polarion test cases and update them in the code?